game/src/main_menu/mainmenu.rs

```diff
@@ -30,30 +30,32 @@ impl Default for MainMenu {
     fn default() -> Self {
         let (sender, receiver) = channel();
 
-        let start_button = Button {
-            text: "Start".to_string(),
-            img: None,
-            message: Start,
-            sender: sender.clone(),
-            rect: graphics::Rect::new(650.0, 180.0, 350.0, 120.0),
-            color: Color::from_rgba(0, 0, 0, 0),
-        };
-        let exit_button = Button {
-            text: "Exit".to_string(),
-            img: None,
-            message: Exit,
-            sender: sender.clone(),
-            rect: graphics::Rect::new(650.0, 420.0, 350.0, 120.0),
-            color: Color::from_rgba(0, 0, 0, 0),
-        };
-        let new_game_button = Button {
-            text: "New Game".to_string(),
-            img: None,
-            message: NewGame,
-            sender: sender.clone(),
-            rect: graphics::Rect::new(650.0, 300.0, 350.0, 120.0),
-            color: Color::from_rgba(0, 0, 0, 0),
-        };
+        let start_button = Button::new(
+            "Start".to_string(),
+            None,
+            Start,
+            sender.clone(),
+            graphics::Rect::new(1322., 350., 350.0, 120.0),
```

Comment:
Gut hier is ne kleine inconsistency mit .0/.

Reply:
Stimmt, das kommt davon, wenn man Werte kopiert...